arch/inst/Zclsd/c.ldsp.yaml

```diff
@@ -0,0 +1,37 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: c.ldsp
+long_name: Stack-pointer based load doubleword to even/odd register pair
+description: |
+  Loads stack-pointer relative 64-bit value into registers rd' and rd'+1. It computes its effective
+  address by adding the zero-extended offset, scaled by 8, to the stack pointer, x2. It expands to ld
+  rd, offset(x2). C.LDSP is only valid when rd≠x0; the code points with rd=x0 are reserved.
```

Comment:
Need to change the content here to match our content with respect to register naming:
- change `rd'` to `rd`
- change `x2` to `sp`